test/plots/barPlotTests.ts

```diff
@@ -1 +1,783 @@
 ///<reference path="../testReference.ts" />
+
+describe("Plots", () => {
+  describe("Bar Plot", () => {
+    describe ("orientations", () => {
+      it("rejects invalid orientations", () => {
+        assert.throws(() => new Plottable.Plots.Bar("diagonal"), Error);
+      });
+
+      it("defaults to vertical", () => {
+        let defaultPlot = new Plottable.Plots.Bar<number, number>();
+        assert.strictEqual(defaultPlot.orientation(), "vertical", "default Plots.Bar() are vertical");
+      });
+
+      it("sets orientation on construction", () => {
+        let verticalPlot = new Plottable.Plots.Bar<number, number>("vertical");
+        assert.strictEqual(verticalPlot.orientation(), "vertical", "vertical Plots.Bar()");
+
+        let horizontalPlot = new Plottable.Plots.Bar<number, number>("horizontal");
+        assert.strictEqual(horizontalPlot.orientation(), "horizontal", "horizontal Plots.Bar()");
```

Comment:
Not worth doing a `forEach()` since the rest of the orientation tests don't need it.